lib/cwflutter.dart

```diff
@@ -63,6 +68,7 @@ class Cwflutter {
     return entities;
   }
 
+  // this method returns all components ordering them by id
```

Comment:
> // this method returns all components ordering them by id

incorrect,
it must be replaced on to:

`// this method returns component by id (by unique identifier)`